src/model/atlas_memory.py

```
@@ -23,7 +23,7 @@
 Architecture (Section 4 "DeepTransformers"):
     1. Project input to key_dim for polynomial expansion
     2. Apply polynomial features: φ(x) = [x, x_i*x_j] for O(d_k²) capacity
-    3. Gated MLP: W1 · (σ(W2·φ(x)) ⊙ W3·φ(x))
+    3. GELU MLP: M(x) = x + W1(σ(W2(φ(x)))) where σ = GELU
```

Ruff (0.14.14)

[warning] Docstring contains ambiguous `σ` (GREEK SMALL LETTER SIGMA). Did you mean `o` (LATIN SMALL LETTER O)?

(RUF002)